master/pkg/model/task_container_defaults.go

```diff
@@ -87,6 +92,12 @@ func (c *TaskContainerDefaultsConfig) Validate() []error {
 	return errs
 }
 
+// KubernetesTaskContainerDefaults is task container defaults specific to Kubernetes.
+// TODO eventually move podSpec here and eventually move this to expconf too.
```

Comment:
I'll fill in JIRA and make tickets for these after review making sure this reasonable

Reply:
makes sense to me.